### PR TITLE
fix: jekyll logic display on Integration page

### LINKED ISSUE #9749: [Docs] Jekyll logic showing through on Integrations page
### Current State

On https://docs.meshery.io/integrations - - 

Screenshot 2023-12-27 at 11 32 23 PM

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by RipulHandoo**
**Notes for Reviewers**

This PR fixes #9749 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0633a2c68e1543639af6737468313919e6a6b3d2 at: https://658d4e5b3174549dfc4ca0ec--meshery-docs-preview.netlify.app

File: docs/pages/integrations/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ abstract: Integrations with other platforms and services.
 
 
 {% assign sorted_index = site.pages | sort: "name" | alphabetical %}
-### All Integrations by Name (forloop.length)
+<!-- ### All Integrations by Name (forloop.length) -->
```
Comment (leecalcote) [inline docs/pages/integrations/index.md:16]: Thanks, however, it would be nice to include a heading with total integration count. Please refactor.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/integrations/index.md (hunk 1 above)
Comment (RipulHandoo) [reply docs/pages/integrations/index.md:16]: Does this code look good to you? Should I refactor code using this?
```
{{% assign sorted_index = site.pages | sort: "name" | alphabetical %}
{% assign integration_count = 0 %}

<!-- All Integrations by Name ({{ site.integrations.size }}) -->
<!-- Count the number of integrations -->
{% for item in sorted_index %}
  {% if item.type == "installation" and item.category == "integrations" and item.list == "include" and item.language == "en" %}
    {% assign integration_count = integration_count | plus: 1 %}
  {% endif %}
{% endfor %}

<!-- Display the total integration count with the message -->
<h2>All Integrations by Name ({{ integration_count }})</h2>
```
```

Comment (leecalcote) [reply docs/pages/integrations/index.md:16]: @RipulHandoo, no, you should use built-ins from liquid templating - https://shopify.github.io/liquid/

**Comment by l5io** [comment]
🚀 Preview for commit e8fe20bc5089fafe3859cd593c55b0a8aedb4c8c at: https://658db54b1f7c2b0fabf0df8f--meshery-docs-preview.netlify.app

File: docs/pages/integrations/index.md (hunk 1 above)
Comment (RipulHandoo) [reply docs/pages/integrations/index.md:16]: Got it. I'll update PR right away.

**Pull request closed without merging**